api/services/ocean/spark/schemas/oceanSparkWorkspace.yaml

```diff
@@ -0,0 +1,39 @@
+type: object
+description: >
+  The object specifying the Ocean Spark workspace.
+title: Ocean Spark Workspace
+required:
+  - oceanWorkspaceId
```

Comment:
I think this `required` list is used to mark the field always returned.
In our case, I think all of them are always returned so they can be listed here